module-2/lab-matplotlib-seaborn/your-code/challenge-3.ipynb

```diff
@@ -9,18 +9,16 @@
   },
```

Comment:
- You can set the params of your plot in just a cell in a similar way as the following: 
```Python
plt.style.use('classic')
sns.set_style('whitegrid')
plt.rcParams['figure.figsize'] = (12, 6)
```
```Python
#- If you run 
titanic['Age'].hist(bins=20)
#instead of 
axes[2].hist(titanic['Age'], bins=50)
#you probably don't print those arrays. Maybe you can try in your notebook. 
- The second way of doing  "Use the appropriate plot to visualize the column Gender. There are 2 ways of doing it. Do it both ways", is not a pie chart, is the countplot plot of seaborn:
```Python
sns.countplot(x='Gender',data=titanic)
```
- What does the previous plot tell you about the feature Age? Comment.
Los rangos intercuartílicos y donde se encuentran la mayoría de los datos en sobre los 22 y los 37 años. El mínimo es dos. El máximo es 80. La mediana es 29. Existen varios outliers por encima. Otra cosa que se puede ver en el boxplot es que la distribución de la edad parece casi simétrica, algo que nos puede confirmar el histograma de frecuencias. 
- When you plot a correlation map as a heatmap is common to adjust the color palette to make it more visual. For example, just one color degraded. You can tune your color palette with cmap parameter.  
- The second way of doing "We suspect that there is a linear relationship between Fare and Age. Use the appropriate plot to show the relationship between these 2 features. There are 2 ways, please do it both ways" may be using a jointplot. 
- ```Python
sns.boxplot(x="Pclass", y="Age", hue="Pclass", data=titanic)
#You can adjust the chart legend position and/or the size of the plot. 
```
- Facetgrid is as simple as the following: 
```Python
g = sns.FacetGrid(data=titanic,col='Gender')
g.map(plt.hist,'Age')
```